packages/test/protos/protobufs.d.ts

```diff
@@ -0,0 +1,186 @@
+import * as $protobuf from "protobufjs";
```

Comment:
I don't think the two generated files will change much, but we could remove from git by changing the `build.proto` script to `pretest`. It adds 1.7s to testing on my machine.

Reply:
IMO you should definitely leave in git and any changes should require a manual regen. This is a testing asset that should not be done on each build IMO.

It is unfortunate that this auto-generated code does not have a top-of-file comment saying it is auto-generated :-(